tutorials/build-golo.md

```diff
@@ -0,0 +1,197 @@
+
+We'll use **[Multipass](https://multipass.run/)** (from **Canonical**) which exists on the leading platforms (Linux, OSX and Windows). Multipass is a lightweight VM manager, simple and easy to use.
+
+> ✋ **Requirements**:
+> - Install Multipass, see the [documentation](https://multipass.run/docs)
+> - Ensure `JAVA_HOME` environment variable is set and points to your JDK installation
+
+## Create the VM and build Golo distribution
+
+- First, create a working directory.
+- In this directory, create two files: `create-vm.sh` and `cloud-init.yaml`.
+- Give the execution rights to `create-vm.sh`.
+
+```shell
+touch create-vm.sh
+touch cloud-init.yaml
+chmod +x create-vm.sh
+```
+
+Add this content to `cloud-init.yaml`
+
+```yaml
+package_update: true
+```
+
+And this is the content of `create-vm.sh`:
+
+```shell
+#!/bin/bash
+
+vm_name="golo-lang-vm"
+vm_cpus=3
+vm_mem="4G"
+vm_disk="100GB"
+
+if [ ! -d "golo-distribution" ];then
+  mkdir golo-distribution
+fi
+
+multipass launch focal --name ${vm_name} --cpus ${vm_cpus} --mem ${vm_mem} --disk ${vm_disk} \
+  --cloud-init ./cloud-init.yaml # 1️⃣
+
+multipass mount golo-distribution ${vm_name}:golo-distribution # 2️⃣
+
+multipass --verbose exec ${vm_name} -- sudo -- bash <<EOF
+apt install openjdk-8-jdk -y # 3️⃣
+apt install -y maven
+apt install -y mercurial
+EOF
+
+multipass --verbose exec ${vm_name} -- bash <<EOF
+git clone https://github.com/eclipse/golo-lang.git
+cd golo-lang
+./gradlew installDist # 4️⃣
+EOF
+
+multipass --verbose exec ${vm_name} -- bash <<EOF
+cp -a golo-lang/build/install/. golo-distribution # 5️⃣
+chmod -R a+rwx golo-distribution
+EOF
+```
+
+> - 1️⃣ creation of the virtual machine
+> - 2️⃣ mounting of a shared directory (between the host and the vm)
+> - 3️⃣ installation of required tools to build Golo
+> - 4️⃣ build the Golo distribution
+> - 5️⃣ copy the distribution to the shared directory
+
+> You can retrieve the source code here: [resources/build-golo/create-vm.sh](resources/build-golo/create-vm.sh)
+
+Now, run the script `create-vm.sh` and wait a few minutes. A moment later you'll get the golo's distribution 🎉 in the shared `golo-distribution` directory:
+
+```shell
+golo-distribution
+└── golo
+   ├── bin
+   │  ├── golo
+   │  ├── golo-debug
+   │  ├── golo-debug.bat
+   │  ├── golo.bat
+   │  ├── golosh
+   │  ├── golosh.bat
+   │  ├── vanilla-golo
+   │  └── vanilla-golo.bat
+```
+
+The last step is to add the `bin` directory of the created directory `golo-distribution/golo` to the PATH environment variable of you host computer.
+
+For example, from your working directory, you can type the following commands (if you use bash):
+
+```shell
+echo "export GOLO_HOME=~/${PWD}/golo-distribution/golo" >> ~/.bashrc
+echo 'export PATH=$PATH:$GOLO_HOME/bin' >> ~/.bashrc
+source ~/.bashrc
+```
+
```

Comment:
For a tutorial, since we maybe don't want to install golo locally, I'd rather do:
```shell
echo "export GOLO_HOME='${PWD}/golo-distribution/golo'" >> ./golo.env
echo 'export PATH=$PATH:$GOLO_HOME/bin' >> ./golo.env
source ./golo.env
```
or `.env` if one wants to use [autoenv](https://github.com/inishchith/autoenv) or similar. 
This env file could even be generated by the `create-vm.sh`script (or not, not sure).
Optionally adding:
```shell
echo "source '${PWD}/golo.env'" >> ~/.bashrc
```
if desired.

Reply:
I don't use `.env`, I had some issues with it (between OSX and Linux)

Reply:
I get the first option